webpack.node.config.ts

```diff
@@ -0,0 +1,72 @@
+import path from 'node:path';
+import { DefinePlugin, type Configuration } from 'webpack';
+
+const entryPoints = {
+  index: './src/index.ts',
+  'electron-main-tracker': './src/lib//electron-main-tracker.ts',
+  'electron-renderer-tracker': './src/lib/electron-renderer-tracker.ts',
+};
+
+const commonConfig: Configuration = {
+  entry: entryPoints,
+  module: {
+    rules: [
+      {
+        test: /\.(ts|tsx)$/,
+        exclude: /node_modules/,
+        use: 'ts-loader',
+      },
+    ],
+  },
+  optimization: {
+    minimize: true,
```

Comment:
I think it'd be ok to not minimize this code given it's a development tool. Minimizing will likely just make debugging and errors harder to read.
```suggestion
    minimize: false,
```

Reply:
Changed it to `false`.